imagevector/containers.yaml

```diff
@@ -405,7 +405,7 @@ images:
   - name: metrics-server
     sourceRepository: github.com/kubernetes-sigs/metrics-server
     repository: registry.k8s.io/metrics-server/metrics-server
-    tag: v0.7.2
+    tag: v0.8.0
```

Comment:
I don't know why renovate raised this PR now. `metrics-server@v0.8.0` is used already but for Kubernetes >= 1.31 only. And `metrics-server@v0.7.2` for Kubernetes < 1.31. See https://github.com/gardener/gardener/blob/7e2ec2cfaf46fad3dd88f4ab678e385fe6525e9c/imagevector/containers.yaml#L387-L422

According to the [Compatibility Matrix](https://github.com/kubernetes-sigs/metrics-server?tab=readme-ov-file#compatibility-matrix) we should continue to use v0.7.2 for Kubernetes < 1.31.

<img width="979" height="417" alt="Screenshot 2025-11-26 at 11 48 09" src="https://github.com/user-attachments/assets/8d8d9457-0015-4545-96a9-80ca4f6c1383" />

---

/close